cosmo/serializer.py

```diff
@@ -628,7 +631,7 @@ def serialize(self):
 
                 l2circuits[l2vpn["name"].replace("WAN: ", "")] = {
                     "interfaces": l2vpn_interfaces,
-                    "description": f"{l2vpn['type']}: " + l2vpn["name"].replace("WAN: ", "") + " via " + remote_device,
+                    "description": f"{l2vpn['type'].upper()}: " + l2vpn["name"].replace("WAN: ", "") + " via " + remote_device,
```

Comment:
why is this one upper()? all the other ones are lower()

Reply:
Because it is only used in the description. 
`l2vpn['type']` is either EPL or epl, depends on Netbox version, but we want EPL as description.